docs/examples/query_engine/sub_question_query_engine.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
since this is going in the docs, would be good to annotate how this works

Reply:
Added notes